en/docs/design/api-security/oauth2/token-persistence.md

```diff
@@ -21,75 +20,43 @@ The flow of synchronous token persistence is as follows:
 3.  If an active access token is found, the token is returned to the client.
 4.  Alternatively, if an existing access token is not found, the OAuth2 component creates a new access token and persists it in the database using the same thread. Once it is persisted, the new token is returned to the client.
 
-!!! tip
-    To enable synchronous token persistence, add following configuration in the `<APIM_HOME>/repository/conf/deployment.toml` to **set the `poolsize` property to 0**.
-
-    ```
-        [session_data]
-        persistence.persistence_pool_size = 0
-    ```
+!!! Info
+    Synchronous token persistence can be enabled by adding the following configuration in the `<API-M_HOME>/repository/conf/deployment.toml` file to **set the `poolsize` property to 0**.
 
+``` toml
+[session_data]
+persistence.persistence_pool_size = 0
+```
 
-#### Asynchronous token persistence
+By default, synchronous token persistence is enabled in WSO2 API Manager 3.0.0. To indicate the number of times to retry in the event of a `CONN_APP_KEY` violation when storing the access token, navigate to file `<APIM_HOME>/repository/conf/deployment.toml` and add the following configuration.
```

Comment:
Shall we change the product version accordingly?

Reply:
oh okay, Thanks

Reply:
fixed